README.md

```diff
@@ -705,8 +705,11 @@ Note that you have 10 minutes to submit your tool output before the run expires.
 
 ### Image Generation
 
-Generate an image using DALL·E! The size of any generated images must be one of `256x256`, `512x512` or `1024x1024` -
-if not specified the image will default to `1024x1024`.
+Generate images using DALL·E 2 or DALL·E 3! 
+
+#### DALL·E 2
```

Comment:
Could ya put these headings in the README index please, at the top?

Reply:
Sure thing, [added](https://github.com/alexrudall/ruby-openai/pull/418/commits/0e86222970e0e077d913d8922cc2663106dcb554)them in now.

Reply:
ta! small typo